addons/account/views/account_account_views.xml

```diff
@@ -59,7 +59,7 @@
                             <page name="accounting" string="Accounting">
                                 <group>
                                     <group>
-                                        <field name="account_type"/>
+                                        <field name="account_type" widget="account_type_selection"/>
```

Comment:
This one does not seems to work due to the fact that there is the same field already defined in invisible a few line below. In my opinion you can delete the <field name="account_type" invisible="1" readonly="1"/> that way it works :)